code/modules/surgery/organs/robolimbs.dm

```diff
@@ -85,28 +100,30 @@ GLOBAL_DATUM(basic_robolimb, /datum/robolimb)
 	parts = list("head")
 	has_subtypes = model
 	is_monitor = TRUE
+	selectable = FALSE
 
 /* Morpheus */
 //Main
 /datum/robolimb/morpheus
+	// This is the Default IPC loadout
 	company = "Morpheus Cyberkinetics"
 	desc = "This limb is simple and functional; no effort has been made to make it look human."
 	icon = 'icons/mob/human_races/cyberlimbs/morpheus/morpheus_main.dmi'
 	has_subtypes = brand
-	is_monitor = TRUE	//Needed for default loadout.
+	is_monitor = TRUE	// Because this one has a SCREEN - it needs to use ipc_face.dm instead of using HAIR.
 
 /datum/robolimb/morpheus/monitor
 	company = "Cyberkinetics Sport"
 	icon = 'icons/mob/human_races/cyberlimbs/morpheus/morpheus_alt1.dmi'
 	parts = list("head")
-	has_subtypes = model //Look, I'm not sure what they're talking about in the original comment below, but this is becoming a model, not childless, so that it will NOT show up in the IPC Synthetic Shell loadout options.
+	has_subtypes = model
 	is_monitor = TRUE
-	//Edge case. We want to be able to pick this one, and if we had it left as null for has_subtypes we'd be assuming it'll be chosen as a child model, and since the parent is unavailable at chargen, we wouldn't be able to see it in the list anyway. Now, we'll be able to select the Morpheus Ckt. Alt. head as a solo-model.
+	selectable = FALSE
```

Comment:
Dangit. This can actually be TRUE. Since both are `is_monitor`, it makes sense that you could change a monitor with a screen to another monitor with a screen. Alas.